app/javascript/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Greeting from './pages/Greetings';
+
+function App() {
+    return (
+        <Greeting/>
+    );
+}
```

Comment:
- [ Required ] Good job implementing this project it works well, however, the project requirements require that you use `react-router` in your `App` component. In the `App` component, you are also required to set up a route that leads to the `Greeting` component so that I don't have to refresh the page to see a new greeting but rather click on the link.

```JavaScript
import React from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import Greeting from './pages/Greetings';

const App = () => {
    return (
      <BrowserRouter>
        <Routes>
           <Route exact path="/greet" element={<Greeting />} />
        </Routes>
      </BrowserRouter>
    );
}

export default App;
```

## Project Requirements
![Screenshot from 2023-03-03 03-17-09](https://user-images.githubusercontent.com/16392046/222602961-19b981da-e1fb-4e59-884a-437682697733.png)